src/drivers/code_hosting/github.sh

```diff
@@ -0,0 +1,16 @@
+#!/usr/bin/env bash
+
+
+function create_pull_request {
+  local repository=$1
+  local branch=$2
+
+  open_browser "https://github.com/$repository/compare/$branch?expand=1"
```

Comment:
That would literally only replace the word `open_browser` with `echo` in them, and cause a bunch of pretty abstract methods in the `activate.sh` file. I don't think that's better or easier than how it is now.